service-commands/.env.dev

```diff
@@ -0,0 +1,9 @@
+IPFS_PORT=5001
```

Comment:
We use dotenv (see `service-commands/src/utils/constants.js`) to import these in when running seed against dev.